ttyplot.c

```diff
@@ -233,15 +247,19 @@ static void show_all_centered(const char *message) {
     mvaddnstr(y, x, message, width);
 }
 
-static int window_big_enough_to_draw(void) {
-    return (width >= 68) && (height >= 5);
+static int window_size_suitable(void) {
+    return (width >= 68) && (height >= 5) && (width < NSAMP);
 }
```

Comment:
If the window size is not suitable, then an error message is displayed by `show_window_size_error()`: “Window too small...”. Now that “not suitable” can also mean “too big”, the error message should be adjusted.